.github/workflows/create-release-branch.yml

```diff
@@ -43,60 +41,38 @@ jobs:
         with:
           node-version: '14.x'
 
-      # Try get node_modules from cache
-      - name: Restore node_modules from cache
-        uses: actions/cache@v2
-        with:
-          path: common/temp/pnpm-store
-          key: ${{ runner.os }}-${{ hashFiles('common/config/rush/pnpm-lock.yaml') }}
-
-      # Install dependencies
-      - name: Install rush
-        run: npm install -g @microsoft/rush@5.47.0
-      - name: Install dependencies
-        run: rush install
-
-      # Bump package versions
-      - name: Bump package versions
-        run: node common/config/node_modules/beachball/bin/beachball bump
-
-      - name: Synchronize package version reported to telemetry
-        run: node common/scripts/sync-telemetry-package-version
-
-      # Important to check version consistency again after bumping versions.
-      - name: Ensure all package versions are consistent
-        run: rush ensure-consistent-versions
-
-      # Retrieve new version to create branch with
+      # get version number from communication react folder
       - name: Retrieve new version from package.json
         id: version
         run: |
           ver=$(jq -r .version packages/communication-react/package.json)
           echo version: $ver
           echo "::set-output name=version::$ver"
 
-      # Commit changes
+      # Create new release branch
       - name: Hop into new branch
         run: git checkout -b release/${{ steps.version.outputs.version }}
+
+      # Choose stable dependency for sdk
```

Comment:
You should look at the branch name of the base branch to decide between stable/beta dependency here.